hash/msx1_flop_525.xml

```diff
@@ -44,18 +44,31 @@ license:CC0-1.0
 	<software name="diskmec2">
 		<description>Disk Mechanic v2.0 (Brazil)</description>
 		<year>1990</year>
-		<publisher>Discovery Informatica</publisher>
+		<publisher>Discovery Informática</publisher>
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="368640">
 				<rom name="disk mechanic v2.0 - discovery informatica.dsk" size="368640" crc="48d7c1d0" sha1="03b80134729235c0febc77becf353a7fd294ad32" />
 			</dataarea>
 		</part>
 	</software>
 
+	<software name="gsystem" supported="no">
+		<description>Gradius System (Brazil)</description>
+		<year>1990</year>
+		<publisher>Nemesis Informática</publisher>
+		<notes>Some menu options ask for other, unavailable, disks.</notes>
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="368640">
+				<rom name="gradius system - nemesis informatica.dsk" size="368640" crc="b06b588d" sha1="66d244adebb62c87c342e3230bc8ca600e189d36" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Should this have “no dump” placeholder parts for the missing disks?  Do we know how many disks there are supposed to be in the full set and what their labels say?

Reply:
The manual for this software does not mention other disks; also, images for this software only show one disk.
Some of the menu options ask for disks named G-COPY or G-CALC; perhaps these are other, unknown, products from Nemesis Informática?

Reply:
Some (all?) of them might not even have been released for all we know, I wouldn't leave documented as "no dump" stuff that's not 100% certain to have existed.